### PR TITLE
Issue #9406: update example of AST for TokenTypes.CASE_GROUP

### LINKED ISSUE #9406: update example of AST for TokenTypes.CASE_GROUP
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CASE_GROUP

Needs to be extended to show example of AST in format that printed by our CLI (change "+-" to "`-") .
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by vivek286**
Closes #9406 

WhatsApp Image 2021-03-16 at 11 45 26 AM

Expected changes for update example of AST for TokenTypes.CASE_GROUP
WhatsApp Image 2021-03-16 at 11 46 09 AM

### CODE REVIEW

**Comment by anshuman10398** [comment]
github, generate website

**Comment by anshuman10398** [comment]
Github, generate web site

**Comment by vivek286** [comment]
@rdiachenko  please review if you have time

**Comment by vivek286** [comment]
@rdiachenko please review if you have time

**Comment by rdiachenko** [comment]
@vivek286 please update description according to https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes (check PR example mentioned)

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -938,35 +938,24 @@
...
-     *             |
-     *             +--ASSIGN (=)
-     *                 |
-     *                 +--IDENT (x)
-     *                 +--NUM_INT (3)
-     *         +--SEMI (;)
-     *         +--LITERAL_BREAK (break)
-     *             |
-     *             +--SEMI (;)
+     * |--CASE_GROUP -&gt; CASE_GROUP
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:941]: remove first `|` in all lines, it is not required

**Review by romani — CHANGES_REQUESTED** [review]
please keep in PR description image of javadoc after code update

**Comment by romani** [comment]
if there will be no reply in a week, this PR be closed.

**Pull request closed without merging**